### PR TITLE
Setup crossword

### PR DESCRIPTION
**Pull request by ella-blaupause**

### CODE REVIEW

File: components/Clue/Clue.test.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { render, screen } from "@testing-library/react";
 import Clue from ".";
 
 test("displays two different symbols", async () => {
-  const randomColor = ["", "red", "blue"];
-  const randomSymbol = ["", "A", "B"];
-  render(<Clue randomColor={randomColor} randomSymbol={randomSymbol} />);
+  const randomColors = ["", "red", "blue"];
+  const randomSymbols = ["", "A", "B"];
+  render(<Clue randomColors={randomColors} randomSymbols={randomSymbols} />);
```
Comment (toshydev) [inline components/Clue/Clue.test.js:7]: Why is this test *async*? You're not using the `userEvent` methods here so it's not necessary, is it?

File: components/Clue/Clue.test.js [hunk 2]
```diff
@@ -13,13 +13,13 @@ test("displays two different symbols", async () => {
...
 
   // find all StyledColorDiv components and check their styles
   const colorDivs = screen.getAllByTestId("color-div");
   expect(colorDivs).toHaveLength(2);
-  expect(colorDivs[0]).toHaveStyle(`background-color: ${randomColor[1]};`);
-  expect(colorDivs[1]).toHaveStyle(`background-color: ${randomColor[2]};`);
+  expect(colorDivs[0]).toHaveStyle(`background-color: ${randomColors[1]};`);
+  expect(colorDivs[1]).toHaveStyle(`background-color: ${randomColors[2]};`);
 });
```
Comment (toshydev) [inline components/Clue/Clue.test.js:25]: Same here. I think `async` is not necessary.

File: components/CrosswordLayout/CrosswordLayout.test.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+import CrosswordLayout from ".";
+
+test("table is correctly rendered with five rows and five columns.", async () => {
```
Comment (toshydev) [inline components/CrosswordLayout/CrosswordLayout.test.js:4]: Same here.
```suggestion
test("table is correctly rendered with five rows and five columns.", () => {
```

**Review by toshydev — APPROVED** [review]
Great work. I left a few suggestions. ✅

File: components/CrosswordLayout/CrosswordLayout.test.js (hunk 3 above)
Comment (ella-blaupause) [reply components/CrosswordLayout/CrosswordLayout.test.js:4]: ✅

File: components/Clue/Clue.test.js (hunk 2 above)
Comment (ella-blaupause) [reply components/Clue/Clue.test.js:25]: ✅

File: components/Clue/Clue.test.js (hunk 1 above)
Comment (ella-blaupause) [reply components/Clue/Clue.test.js:7]: If I remove async and await here, the test fails?🤨

**Pull request merged**